src/app/(post-login)/submission/program/page.tsx

```diff
@@ -24,7 +24,7 @@ import PROGRAMS_LIST_QUERY from '@/app/gql/PROGRAMS_LIST_QUERY';
 import { notNull } from '@/global/utils/types';
 import { useQuery } from '@apollo/client';
 import { notFound } from 'next/navigation';
-import ProgramList from './components/ProgramList';
+import ProgramList from '../components/ProgramList';
```

Comment:
For consistency...
```suggestion
import ProgramList from '@/components/ProgramList';
```

Reply:
This is correct.it is confusing organisation pattern.
Leaning heavily into Next colocation a lot of pages have a `components` folder for specific uses of a component but there's also a components folder in the root at `@/components` for general use

Reply:
wait, so you mean there's a `src/app/(post-login)/submission/components`? 👎

Reply:
that's exactly what I mean.
any suggestions on improving that organisation?

Reply:
one option is to remove the "component" folder but then you have a bunch of uncatagorised files in a single folder